es6-classes/square.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super(width);
```

Comment:
You should be passing in 2 arguments here. One representing the area, and the other the perimeter